components/reconciler/makefile

```diff
@@ -1,12 +1,12 @@
 OAPI_GENERATOR=oapi-codegen
 OAPI_FILE=./docs/external_api.yaml
-OAPI_REVISION=05b7b7a4d5d503972dcfb2a6d1397ee15cf6307f
+OAPI_REVISION=modify-reconciliation-info-endpoint
 
 CLIENT_FILE=./pkg/client_gen.go
 TYPES_FILE=./pkg/types_gen.go
 
 docs/external_api.yaml:
-	curl -o $(OAPI_FILE) --create-dirs https://raw.githubusercontent.com/kyma-incubator/reconciler/$(OAPI_REVISION)/openapi/external_api.yaml
+	curl -o $(OAPI_FILE) --create-dirs https://raw.githubusercontent.com/dbadura/reconciler/$(OAPI_REVISION)/openapi/external_api.yaml
```

Comment:
remember to revert these changes after merging [this PR](https://github.com/kyma-incubator/reconciler/pull/427) :)